lisa_commands/commands.py

```diff
@@ -110,6 +110,17 @@ def commands():
             'function': youtube_me,
             "description": "use text after command to query youtube"
         },
+        'spotify me': {
```

Comment:
nit I try to keep this list and the function it maps to in alphabetical order.
So spotify me should probably go after `shame`